src/features/user/TreeMapper/Analytics/components/Map/components/PlantLocationDetails/index.tsx

```diff
@@ -0,0 +1,120 @@
+import { useTranslation } from 'react-i18next';
+import styles from './index.module.scss';
+import {
+  PlantLocationDetailsApiResponse,
+  PlantLocation,
+} from '../../../../../../../common/types/dataExplorer';
+import PlantLocationDetailsZeroState from '../PlantLocationDetailsZeroState';
+import TreeMapperIcon from '../TreeMapperIcon';
+
+interface Props {
+  plantLocationDetails: PlantLocationDetailsApiResponse['res'] | null;
+  selectedLayer: PlantLocation['properties'];
+}
+
+const PlantLocationDetails = ({
+  plantLocationDetails,
+  selectedLayer,
+}: Props) => {
+  const { t } = useTranslation(['treemapperAnalytics']);
+
+  return (
+    <div className={styles.plantLocationDetailsContainer}>
+      <div className={styles.content}>
+        {plantLocationDetails ? (
+          <div className={styles.contentTop}>
+            <div className={styles.topContainer}>
+              <div className={styles.leftContainer}>
+                <p className={styles.title}>{t('speciesPlanted')}</p>
+                <p>
+                  {selectedLayer.treeCount}&nbsp;{t('trees')}
+                </p>
+              </div>
+              <div className={styles.rightContainer}>
+                <div className={styles.title}>
+                  <p>{t('plantingDensity')}</p>
+                </div>
+                <p>
+                  {selectedLayer?.density?.toFixed(4)}&nbsp;
+                  {t('treesPerHa')}
+                </p>
+              </div>
+            </div>
+            <div className={styles.midContainer}>
+              <div className={styles.title}>
+                {t('speciesPlanted')}&nbsp;(
+                {plantLocationDetails.plantedSpecies.length})
```

Comment:
Testing this with my test account I get a `TypeError: r.plantedSpecies is null` in the browser console when clicking on one of my sample tree registrations:
<img width="1473" alt="Bildschirmfoto 2024-02-28 um 10 01 10" src="https://github.com/Plant-for-the-Planet-org/planet-webapp/assets/1532418/388423f5-18f3-4209-b102-386c7e53961b">

Reply:
@norbertschuler I'm unable to reproduce this, could you see if it's still an issue?

Reply:
I have to create a new TPO project first and add some trees as they got lost after copying prod to stage, but so I saw the page displays an endless spinner of no project exists for a TPO and crashes if you have a new project without any items if you select it on the map. Both edge cases have not properly been handled by the code.

Reply:
So, yes, I am still seeing a crash for my empty project:
<img width="1470" alt="Bildschirmfoto 2024-03-10 um 12 45 38" src="https://github.com/Plant-for-the-Planet-org/planet-webapp/assets/1532418/50a7654e-7f5e-422d-884b-94c7f321e3e1">

Reply:
Has been resolved now